web_external/vue/views/AnnotationTool.vue

```diff
@@ -0,0 +1,280 @@
+                window.location.replace(redirectUrl);
+            }
+        }
+    },
+    created() {
+    },
+    mounted() {
+        this.user = getCurrentUser();
+        this.getAnnotation({id: this.annotationId});
+    },
+    methods: Object.assign({
+        reset() {
+            if (this.activeFeatureId) {
+                this.deactivateFeature(this.activeFeatureId);
+            }
+            this.setActiveFeatureId(null);
+            this.setShowReview(false);
+            this.setMarkupState(MarkupState.DEFINITE);
+            this.resetResponses();
+            this.resetMarkups();
+        },
+        resetMarkups() {
+            this.setMarkups({});
+        },
+        resetResponses() {
+            const responses = {};
+            this.questions.forEach((question) => {
+                responses[question.id] = null;
+            });
+            this.setResponses(responses);
+        },
+        deactivateFeature(featureId) {
+            // Save feature markup
+            // Optimization: freeze array so that it isn't reactive
+            const values = this.$refs.viewer.getActiveValues();
+            this.setMarkup({
+                featureId: featureId,
+                values: Object.freeze(values)
+            });
+            this.$refs.viewer.clear();
+
+            this.setActiveFeatureId(null);
+        },
+        onFeatureActivated(featureId) {
+            if (this.activeFeatureId) {
+                this.deactivateFeature(this.activeFeatureId);
+            }
+            this.setActiveFeatureId(featureId);
+
+            let markup = null;
+            if (this.markups.hasOwnProperty(featureId)) {
+                // Copy frozen array to allow editing
+                markup = this.markups[featureId].slice(0);
+            }
+            this.$refs.viewer.activate(markup);
+        },
+        onFeatureDeactivated(featureId) {
+            this.deactivateFeature(featureId);
+        },
+        onDisplayFeature(featureId) {
+            if (featureId) {
+                const markup = this.markups[featureId];
+                this.$refs.viewer.display(markup);
+            } else {
+                this.$refs.viewer.clear();
+            }
+        },
+        onDeleteFeature(featureId) {
+            if (this.activeFeatureId === featureId) {
+                this.deactivateFeature(featureId);
+            }
+
+            this.setMarkup({
+                featureId: featureId,
+                values: null
+            });
+        }
+    }, mapMutations([
+        'setFlagStatus',
+        'setShowReview',
+        'setMarkupState',
+        'setResponses',
+        'setMarkups',
+        'setMarkup',
+        'setActiveFeatureId'
+    ]), mapActions([
+        'getAnnotation',
+        'submitAnnotation'
+    ]))
+};
+</script>
+
+<style lang="stylus" scoped>
+.annotation-tool-container
+  background #000
```

Comment:
We don't need to override with a black background for any of these components. We can inherit the color scheme of the main site, for the most part.

Reply:
I preserved the old look and feel to focus on the functionality instead of a redesign. I think inheriting the white site background forces use to reconsider the style of everything to be usable.